README.md

```diff
@@ -29,23 +31,45 @@ config.middleware.use "Klaviyo::Client::Middleware", "YOUR_KLAVIYO_API_TOKEN"
 
 This will automatically insert the Klaviyo script at the bottom on your HTML page, right before the closing `body` tag.
 
-To record customer actions directly from your backend, in your `application_controller` class add a method to initialize Klaviyo.
+To record customer actions directly from your backend, in your `application_controller` class add a method to initialize set your public and private API tokens.
 
 ```ruby
-class ApplicationController < ActionController::Base
-  before_filter :initialize_klaviyo
+require 'klaviyo'
```

Comment:
please update to show all methods in the readme, also please do a local install of the gem to make sure these are correct vs using pry to envoke Ex. should it be Klaviyo::Lists.get_list()